modules/dev-tools/assets/js/editor/dev-tools.js

```diff
@@ -1,14 +1,102 @@
+/**
+ * @typedef {Object} Version
+ * @property {number} major1
+ * @property {number} major2
+ * @property {number} minor
+ * @property {string} build
+ */
 export default class extends elementorModules.editor.utils.Module {
-	onElementorLoaded() {
-		this.notifyDeprecated();
-	}
-
-	notifyDeprecated() {
+	notifyBackendDeprecations() {
 		// eslint-disable-next-line camelcase
 		const notices = elementor.config.dev_tools.deprecation.soft_notices;
 
 		Object.entries( notices ).forEach( ( [ key, notice ] ) => {
 			elementorCommon.helpers.softDeprecated( key, ...notice );
 		} );
 	}
+
+	/**
+	 * @param {string} version
+	 *
+	 * @return {Version}
+	 */
+	parseVersion( version ) {
+		const versionParts = version.split( '.' ),
+			versionPartsLength = versionParts.length;
```

Comment:
It's almost the exact same length of a string 😅 
I think it's redundant
```
versionPartsLength
versionParts.length
```

Reply:
Handled.